envi/archs/arm/disasm.py

```diff
@@ -2083,27 +2140,30 @@ def getOperAddr(self, op, emu=None):
         # there are certain circumstances where we can survive without an emulator
         pubwl = self.pubwl >> 3
         u = pubwl & 1
-
+        tsize = self.tsize # to help cope with ldcl
```

Comment:
merging two branches which modify the same code will end in conflict.  conflict isn't necessarily bad.  here's some help on git and conflicts.

https://help.github.com/articles/resolving-a-merge-conflict-from-the-command-line/
https://githowto.com/resolving_conflicts